### DISCUSSION TITLE
macOS support for RN 0.68+?

### DISCUSSION THREAD

**Discussion by faridsaud** [microsoft/react-native-windows#9961] [General]
Hi,

I was going through the documentation for [Get Started with macOS](https://microsoft.github.io/react-native-windows/docs/rnm-getting-started) and I could notice the following statement:
```
** Latest stable version available for React Native for macOS is 0.64** Please visit the [0.64 version of the Getting Started guide](https://microsoft.github.io/react-native-windows/docs/0.64/rnm-getting-started) to build React Native for macOS apps.
```

Are there plans to keep adding support for the latest React Native versions for macOS?

**Comment by Saadnajmi** [comment]
Hello, could you make an issue in the React Native macOS repo? https://github.com/microsoft/react-native-macos/issues/new/choose

Quick answer, we've released React Native macOS 0.66 recently, and 0.68 is coming imminently :)
